ml/src/main/scala/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.scala

```diff
@@ -37,6 +39,8 @@ trait H2ODeepLearningParams extends H2OAlgoSupervisedParams[DeepLearningParamete
   private val l2 = doubleParam("l2")
   private val hidden = intArrayParam("hidden")
   private val reproducible = booleanParam("reproducible")
+  private val activation = stringParam("activation")
```

Comment:
Is there any enum on H2O side for activation functions?

Reply:
yes, we are using it :) But on scala side we represent enum values as string

Reply:
ok, thanks!